portality/api/current/bulk/articles.py

```diff
@@ -47,7 +47,7 @@ def create(cls, articles, account):
             raise Api401Error()
 
         # convert the data into a suitable article models
-        articles = [ArticlesCrudApi.prep_article(data, account) for data in articles]
+        articles = [ArticlesCrudApi.prep_article_for_api(data, account) for data in articles]
```

Comment:
This seems like an slightly confusing method name change.  The article here isn't being prepped *for* the API, it's being prepped for the main codebase, so if we change the method name at all it should be `prep_ariticle_for_core` or something?  I'm not sure, I would probably just leave it as it was.

Reply:
the reason call it `_for_api` is because this method for api layer only because it will throw `Api400Error`

the change  extracted  `ArticlesCrudApi.prep_article` for reuseable in both api and background job. 
may be we should rename `ArticlesCrudApi.prep_article` or move `prep_article` to other class, any suggestion?